src/main/java/org/tikv/common/util/ConcreteBackOffer.java

```diff
@@ -185,6 +184,7 @@ public boolean canRetryAfterSleep(BackOffFunction.BackOffFuncType funcType, long
       }
     }
 
+    SlowLogSpan slowLogSpan = getSlowLog().start("backoff " + funcType.name());
```

Comment:
why move it here? should `BACKOFF_DURATION` also be moved?

Reply:
there is a `return` in line 184, which will cause ` "end":"N/A",`.
it's hard to understand it.

Reply:
got. then why not move `BACKOFF_DURATION` to here as well?

Reply:
good idea

Reply:
Moving these lines to here might cause we can't see the backoff in slow log.

Reply:
We may end this span while returning though we will see this backoff end shortly but this span is still there.